common/changes/office-ui-fabric-react/choicegroup_classname_2018-11-13-19-36.json

```
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Removing label's setting to classname beacuse it's a child element and shouldn't have the parent choicegroup's classname applied to it.",
```

```suggestion
      "comment": "Choicegroup: Fix an issue where the child label component was incorrectly getting the className prop.",
```

You have a typo in `because` :)